src/Compiler.php

```diff
@@ -186,7 +180,7 @@ public function getDebugInfo(): array
     {
         ksort($this->debugInfo);
 
-        return $this->debugInfo;
+        return $this->debugInfo ?? [];
```

Comment:
If debugInfo is not an array, the call to ksort would fail.

Reply:
Do we have data in debuginfo that is not available as an array, although the function :: requires an array ?

Reply:
debugInfo is always an array.